tests/NLog.UnitTests/Targets/DatabaseTargetTests.cs

```diff
@@ -607,7 +607,13 @@ public void LevelParameterTest(DbType? dbType, bool noRawValue, string expectedV
         [InlineData("${event-properties:almostAsIntProp}", DbType.Int32, 124)]
         [InlineData("${event-properties:almostAsIntProp}", DbType.Int64, (long)124)]
         [InlineData("${event-properties:almostAsIntProp}", DbType.AnsiString, " 124 ")]
-        public void GetParameterValueTest(string layout, DbType dbtype, object expected, bool convertToDecimal = false)
+        [InlineData("${event-properties:NullRawValue}", DbType.AnsiString, "")]
+        [InlineData("${event-properties:NullawValue}", DbType.Int32, 0)]
+        [InlineData("${event-properties:NullRawValue}", DbType.AnsiString, null, true)]
+        [InlineData("${event-properties:NullRawValue}", DbType.Int32, null, true)]
```

Comment:
It would be nice if you could add tests for Guid as well:

```suggestion
        [InlineData("${event-properties:NullRawValue}", DbType.Int32, null, true)]
        [InlineData("${event-properties:NullRawValue}", DbType.Guid, null, true)]
```